packages/@sanity/desk-tool/src/DeskTool.test.tsx

```diff
@@ -0,0 +1,551 @@
+/* eslint-disable max-nested-callbacks */
+/// <reference types="@sanity/types/parts" />
+
+import * as Rx from 'rxjs'
+import * as operators from 'rxjs/operators'
+import {render, act} from '@testing-library/react'
+import React, {useEffect, useMemo, useState} from 'react'
+import {useRouter} from '@sanity/base/router'
+import {RouterProvider} from '@sanity/state-router/components'
```

Comment:
Should `RouterProvider` be imported from `@sanity/base` here? Then we don’t have to depend on state-router in desk-tool (and have more parity, I believe).

```suggestion
import {RouterProvider, useRouter} from '@sanity/base/router'
```